src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerExtensions.cs

```diff
@@ -20,9 +20,13 @@ public static class LoggerExtensions
         /// <param name="logger">The <see cref="ILogger"/> to write to.</param>
         /// <param name="eventId">The event id associated with the log.</param>
         /// <param name="exception">The exception to log.</param>
-        /// <param name="message">Format string of the log message in message template format. Example: <c>"User {User} logged in from {Address}"</c></param>
+        /// <param name="message">Format string of the log message in message template format. Example: <c>"User {User} logged in from {Address}"</c>.</param>
         /// <param name="args">An object array that contains zero or more objects to format.</param>
-        /// <example>logger.LogDebug(0, exception, "Error while processing request from {Address}", address)</example>
+        /// <example>
+        /// <code language="csharp">
```

Comment:
Good to know the `<code>` needs to be specified within `<example>`. I wasn't aware.